docs/storage_options.md

```diff
@@ -60,9 +62,37 @@ signature_version = s3v4
 S3 Bandersnatch mirrors are designed to be served with s3 static sites and
 can also be used with the Amazon CDN service or another CDN service.
 
+I assume you have already set up an AWS account and S3 bucket, and the Bandersnatch sync job has been proceeded.
+
+### Enabling website hosting for the bucket
+When you enable the website hosting for a bucket, this bucket can be viewed as static website. Using the s3 domain or your customized domain.
```

Comment:
NIT - Markdown wants a \n between all headings - Please do to others.

- I need to add prettier auto formattting to our .md files ...

```suggestion
### Enabling website hosting for the bucket

When you enable the website hosting for a bucket, this bucket can be viewed as static website. Using the s3 domain or your customized domain.
```